docker/images/dev-docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Creates an image for skycoin development
+FROM skycoin/skycoindev-cli:develop
+
+# Install docker
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+    btrfs-progs \
+    e2fsprogs \
+    iptables \
+    xfsprogs \
+    apt-transport-https \
+    ca-certificates \
+    curl \
```

Comment:
`curl` is already installed on the base image. Please check if there are other packages of this list in the same situation.